devito/profiler.py

```diff
@@ -81,18 +86,23 @@ def add_profiling(self, code, name, byte_size=4, omp_flag=None):
                           Defaults to 4.
         :param omp_flag: OpenMP flag to add before profiling operations,
                          if needed.
+        :param to_ignore: List of strings containing the labels of
+                          symbols used as loop variables
```

Comment:
While it's probably possible to extract some loop variables from the `cgen.For` objects, some others (like the time steppers) are not declared in the For statement and need to be passed in using `to_ignore`. 

We might as well have them all passed in `to_ignore`